app/recordtransfer/static/recordtransfer/css/base/tooltip.css

```
@@ -0,0 +1,43 @@
+.form-tooltip {
+    /* Float on top of the UI */
+    position: absolute;
+    display: none;
+
+    /* Avoid layout interference */
+    width: max-content;
+    top: 0;
+    left: 0;
+
+    /* Fade in/out */
+    opacity: 0;
+    transition: opacity 0.2s ease-in-out;
+
+    padding: 6px;
+    font-size: 11pt;
+    background-color: #333333;
+    color: white;
+    border: 5px;
+    border-radius: 5px;
+}
+
+.help-tooltip.help-icon {
+    z-index: 1000;
+}
```

One last thing, can you delete the `.help-icon` class from `submission_form.css` and move its styles here?

```suggestion
.help-tooltip.help-icon {
    z-index: 1000;
    margin-left: 10px;
}
```